doc/classes/Control.xml

```diff
@@ -142,13 +142,13 @@
 				}
 				[/csharp]
 				[/codeblocks]
-				The event won't trigger if:
-				* clicking outside the control (see [method _has_point]);
-				* control has [member mouse_filter] set to [constant MOUSE_FILTER_IGNORE];
-				* control is obstructed by another [Control] on top of it, which doesn't have [member mouse_filter] set to [constant MOUSE_FILTER_IGNORE];
-				* control's parent has [member mouse_filter] set to [constant MOUSE_FILTER_STOP] or has accepted the event;
-				* it happens outside the parent's rectangle and the parent has either [member clip_contents] enabled.
-				[b]Note:[/b] Event position is relative to the control origin.
+				If the [param event] inherits [InputEventMouse], this method will [b]not[/b] be called when:
+				- the control's [member mouse_filter] is set to [constant MOUSE_FILTER_IGNORE];
+				- the control is obstructed by another control on top, that doesn't have [member mouse_filter] set to [constant MOUSE_FILTER_IGNORE];
+				- the control's parent has [member mouse_filter] set to [constant MOUSE_FILTER_STOP] or has accepted the event;
+				- the control's parent has [member clip_contents] enabled and the [param event]'s position is outside the parent's rectangle;
+				- the [param event]'s position is outside the control (see [method _has_point]).
+				[b]Note:[/b] The [param event]'s position is relative to this control's origin.
```

Comment:
Not exactly happy about this, still, but changing it too much would be way out of scope.